server/leafnode.go

```diff
@@ -509,6 +514,8 @@ func (s *Server) connectToRemoteLeafNode(remote *leafNodeCfg, firstConnect bool)
 			case <-s.quitCh:
 				return
 			case <-time.After(reconnectDelay):
+				// Check if we should migrate any JetStream assets while this remote is down.
+				s.checkJetStreamMigrate(remote)
```

Comment:
But if we are not connected to the rest of the network, how can migration work?

Reply:
Check the issue referenced. We are just moving the leader such that the other side can access again.

Reply:
Maybe should rename, just migrating the leaders.